tests/CI/docker-compose.yml

```
@@ -1,5 +1,11 @@
 version: '3.4'
 
+volumes:
+  # Volume used to store the config of diracx
+  diracx-cs-store:
+  # Volume used to store the pair of keys to sign the tokens
+  diracx-key-store:
+
```

This feels like it should not have the semi-colons...?

```suggestion
volumes:
  # Volume used to store the config of diracx
  diracx-cs-store
  # Volume used to store the pair of keys to sign the tokens
  diracx-key-store
```

---

It's correct: https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes